index.html

```diff
@@ -0,0 +1,193 @@
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.1/css/all.css" integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf" crossorigin="anonymous">
+  </head>
+
+  <body class="bg-light">
+    <header id="header">
+      <nav class="navbar navbar-expand-lg fixed-top navbar-light bg-light" id="nav-bar">
+        <div class="container">
+          <a class="navbar-brand" href="#">
+                <img src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" width="30%" height="60%" alt="" id="header-img">
+              </a>
+          <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive" aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+              </button>
+          <div class="collapse navbar-collapse" id="navbarResponsive">
+            <ul class="navbar-nav ml-auto">
+              <li class="nav-item">
+                <a class="nav-link" href="#features">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#works">How it works</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#pricing">Pricing</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <div class="row" style="padding-top: 80px;">
+      <div class="col-12">
+        <h4 style="text-align: center;">
+          <b>Handcrafted, home-made masterpieces</b>
+        </h4>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-4"></div>
+      <div class="col-4">
+        <form id="form" method="POST" action="https://www.freecodecamp.com/email-submit">
+          <div class="form-group">
+            <input type="email" class="form-control" name="email" id="email" aria-describedby="emailHelp" placeholder="Enter email" required>
+          </div> <p></p><p></p>
+          <div class="form-group">
+            <div style="text-align: center;">
+              <input type="submit" id="submit" class="btn btn-warning text-center"  value="GET STARTED"/>
+            </div>
+          </div>
+        </form>
+      </div>
+      <div class="col-4"></div>
+    </div> 
+    <div class="row" style="padding-top: 100px;">
+      <div class="row">
+          <div class="col-4" style="padding-right: 90px;">
+            <i class="fas fa-fire" style="font-size:xxx-large; color: orange; float: right;"></i>
+
+          </div>
+          <div class="col-7">
+              <div>
+                <h4><b> Premium Materials</b></h4>
+                Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.
```

Comment:
Let's use css to increase the font-weight of the text when needed ok! You are doing a great job so far. But to follow proper semantics and keep uphold separation of concerns, inline css and having `<b>` tag inside of the `<h4>` tag can be misread 👏🏾

Reply:
Nice review. Very polite, well explained, and human-friendly due to the use of emojis